dynamo/configuration.py

```diff
@@ -4,16 +4,32 @@
 from cycler import cycler
 import matplotlib.pyplot as plt
 
-# set the data store mode.
-# saving memory or storing more results
-# modes: verbose, succint
-data_store_mode = "verbose"
-keep_filtered_genes = True
-keep_raw_layers = True
-
-if data_store_mode == "succint":
-    keep_filtered_genes = False
-    keep_raw_layers = False
+
+class DynamoConfig:
+    # set the data store mode.
+    # saving memory or storing more results
+    # modes: verbose, succint
+    data_store_mode = "verbose"
```

Comment:
maybe rename verbose to full. this is not about a lot of words (verbose's meaning)